bigbluebutton-html5/private/config/settings.yml

```diff
@@ -41,6 +41,9 @@ public:
       dataSaving:
         viewParticipantsWebcams: true
         viewScreenshare: true
+      webRTCStats:
+        logging: true
```

Comment:
It should be disabled by default. That's going to be a lot of noise that people don't need.